enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/infotrygd/InfotrygdPerioder.kt

```diff
@@ -9,10 +10,64 @@ data class InfotrygdPerioderArenaRequest(val personIdenter: Set<String>,
                                          val fomDato: LocalDate? = null,
                                          val tomDato: LocalDate? = null)
 
-data class InfotrygdPerioderResponse(val perioder: List<InfotrygdPeriode>)
+data class InfotrygdPerioderArenaResponse(val perioder: List<InfotrygdArenaPeriode>)
+
+data class InfotrygdArenaPeriode(val personIdent: String,
+                                 val fomDato: LocalDate,
+                                 val tomDato: LocalDate,
+                                 val beløp: Float,
+                                 val opphørsdato: LocalDate? = null)
+
+data class InfotrygdPeriodeRequest(val personIdenter: Set<String>,
+                                   val stønadstyper: Set<StønadType>)
+
+data class InfotrygdPeriodeResponse(val perioder: List<InfotrygdPeriode>)
 
 data class InfotrygdPeriode(val personIdent: String,
-                            val fomDato: LocalDate,
-                            val tomDato: LocalDate,
-                            val beløp: Float,
-                            val opphørsdato: LocalDate? = null)
\ No newline at end of file
+                            val kode: InfotrygdEndringKode,
+                            val brukerId: String,
+                            val stønadId: Long,
+                            val vedtakId: Long,
+                            val stønadBeløp: Int,
+                            val inntektsreduksjon: Int,
+                            val samordningsfradrag: Int,
+                            val beløp: Int,
+                            val startDato: LocalDate,
+                            val stønadFom: LocalDate,
+                            val stønadTom: LocalDate,
+                            val opphørsdato: LocalDate?)
+
+/**
+ * Per 2021-09-30 finnes kun noen av disse i databasen:
+ * AN
+ * E
+ * F
+ * G
+ * NY
+ * O
+ * UA
+ */
+enum class InfotrygdEndringKode(val beskrivelse: String) {
```

Comment:
Litt usikker på om vi trenger alle disse, eller om vi bare skal ta med de som nevnes att de finnes i databasen, plus `OO` som kommer bli aktuell for oss. Har spurt infotrygd om det.